internal/pkg/template/workload.go

```diff
@@ -324,12 +324,20 @@ type CapacityProviderStrategy struct {
 	CapacityProvider string
 }
 
+// Cooldown holds configuration needed for autoscaling cooldown fields.
+type Cooldown struct {
+	ScaleInCooldown  *float64
+	ScaleOutCooldown *float64
+}
+
 // AutoscalingOpts holds configuration that's needed for Auto Scaling.
 type AutoscalingOpts struct {
 	MinCapacity  *int
 	MaxCapacity  *int
 	CPU          *float64
+	CPUCooldown  *Cooldown
```

Comment:
i think this and `MemCooldown` don't need to be pointers, which could simplify the template/transform logic